index.html

```diff
@@ -7,6 +7,7 @@
     <script type="text/javascript" src="javascripts/2048.js"></script>
   </head>
   <body>
+    <h6>New Game</h6>
```

Comment:
It's not really obvious to the player that you would need to click on this H6 to start a game. You should consider using a button-style link